lua/neogit/popups/commit/actions.lua

```diff
@@ -33,13 +33,18 @@ local function confirm_modifications()
 end
 
 local function do_commit(popup, cmd)
-  client.wrap(cmd.arg_list(popup:get_arguments()), {
+  cmd.arg_list(popup:get_arguments())
```

Comment:
Have you tried just adding an option for `-m` to the commit popup?
You could do `popup:option("m", "message", "", "Commit message", { key_prefix = "-" })` and that also just bypasses the editor.	

https://github.com/NeogitOrg/neogit/blob/d2a2ae4415872fbddb2441920ab109ee52fd4916/doc/neogit.txt#L2267-L2283